languages/python/sqlalchemy-oso/tests/test_roles2.py

```diff
@@ -222,7 +222,7 @@ def test_empty_role(init_oso):
     # defining role with no permissions/implications throws an error
     oso, session = init_oso
     policy = """
-    resource(_type: Organization, "org", actions, roles) if
+    resource(Organization, "org", actions, roles) if
```

Comment:
Wondering if we should switch the order here so that the name comes first. What do others think?

Reply:
Yeah that seems like a good idea.